codec-http2/src/main/java/io/netty/handler/codec/http2/internal/hpack/Encoder.java

```diff
@@ -351,10 +360,6 @@ private void add(byte[] name, byte[] value) {
             remove();
         }
 
-        // Copy name and value that modifications of original do not affect the dynamic table.
```

Comment:
true but I would say if a user puts a StringBuilder into our http headers we are kind of f**ed anyway as the if the content changes the hash will not be correct anymore.